tests/boards/nrf52_bsim/egu/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_ZTEST=y
+CONFIG_TEST=y
```

Comment:
TEST is selected by [ZTEST](https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/testsuite/ztest/Kconfig)